pepper-apis/src/main/java/org/broadinstitute/ddp/copy/CopyExecutor.java

```diff
@@ -125,6 +125,17 @@ private void copyAnswersFromPreviousInstance(Handle handle, CopyConfiguration co
         for (var childStableIds : childQuestionStableIds.values()) {
             questionStableIds.addAll(childStableIds);
         }
+
+        Set<String> specifiedStableIds = config.getPreviousInstanceFilters().stream()
```

Comment:
How is the execution_order retained from here on !

Reply:
Thanks for the catch! It wasn't respecting execution order of these filters but I have refactored to do so!

Reply:
:+1: